src/BaGet.Core/Indexing/SymbolIndexingService.cs

```diff
@@ -110,10 +141,9 @@ public async Task<SymbolIndexingResult> IndexAsync(Stream stream, CancellationTo
 
                 return files.Where(p => Path.GetExtension(p) == ".pdb").ToList();
             }
-            catch (Exception)
+            catch (Exception e)
             {
-                // TODO: ValidatePackageEntries throws PackagingException.
-                // Add better logging.
+                _logger.LogError(e, "Unable to get symbol package Pdb paths due to exception");
```

Comment:
Nice log statement!

Could you re-add the `// TODO: ValidatePackageEntries throws PackagingException.` comment? I *think* we should return `SymbolIndexingResult.InvalidSymbolPackage` in that case, but it's been a while since I've looked at this path.